hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/utils/ControllerIdHtmlEncoder.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2022 Bosch.IO GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.ui.utils;
+
+import java.util.Base64;
+import java.util.Base64.Encoder;
+import java.util.Objects;
+
+/**
+ * Encodes controller IDs to make them embeddable into HTML as element
+ * identifiers.
+ */
+public class ControllerIdHtmlEncoder {
```

Comment:
Weird.. this class is already present. Why is it showing it as a new added class?